elements/pfe-tabs/src/pfe-tab-panel.scss

```
@@ -70,7 +70,18 @@ $LOCAL-VARIABLES: (focus: pfe-var(link),
 /// HORIZONTAL
 /// ===========================================================================
 :host([variant="earth"]) {
+  background-color: pfe-local(BackgroundColor, $region: panel, $fallback: #{pfe-var(surface--lightest)});
```

```suggestion
  background-color: pfe-local(BackgroundColor, $region: panel, $fallback: pfe-var(surface--lightest));
```

You shouldn't need the interpolation here since it's being assigned to a variable, it will automatically resolve the function call.